src/modules/dashboard/components/UnlockTokenDialog/UnlockTokenDialog.tsx

```diff
@@ -55,23 +69,37 @@ const UnlockTokenDialog = ({
   return (
     <ActionForm
       initialValues={{
+        forceAction: false,
         annotationMessage: undefined,
+        /*
+         * @NOTE That since this a root motion, and we don't actually make use
+         * of the motion domain selected (it's disabled), we don't need to actually
+         * pass the value over to the motion, since it will always be 1
+         */
       }}
-      submit={ActionTypes.COLONY_ACTION_UNLOCK_TOKEN}
-      error={ActionTypes.COLONY_ACTION_UNLOCK_TOKEN_ERROR}
-      success={ActionTypes.COLONY_ACTION_UNLOCK_TOKEN_SUCCESS}
+      validationSchema={validationSchema}
+      submit={getFormAction('SUBMIT')}
+      error={getFormAction('ERROR')}
+      success={getFormAction('SUCCESS')}
       onSuccess={close}
       transform={transform}
     >
-      {(formValues: FormikProps<any>) => (
-        <Dialog cancel={cancel}>
-          <UnlockTokenForm
-            {...formValues}
-            colony={colony}
-            back={prevStep && callStep ? () => callStep(prevStep) : undefined}
-          />
-        </Dialog>
-      )}
+      {(formValues: FormikProps<FormValues>) => {
+        if (formValues.values.forceAction !== isForce) {
+          setIsForce(formValues.values.forceAction);
```

Comment:
This is an error that seems to be with all `force` toggle switches. It probably needs to be wrapped in an effect. Do you think this is worth creating another issue for?

![formik-setState-error](https://user-images.githubusercontent.com/33682027/148536627-17c2156b-adc7-4784-b7cc-4855c4cca6aa.jpg)

Reply:
It's not an error, it's just a warning and it's present in other dialogs as well (as per example) as they are all done in a similar way
<img width="1252" alt="Screenshot 2022-01-08 at 12 26 44" src="https://user-images.githubusercontent.com/34057551/148653665-8bad8a2a-4606-4981-aabd-84836bd2d9a4.png">
 
It might be worth creating an issue but with a low priority as it's only a warning.

Reply:
Thanks @chinins